src/render/index.ts

```diff
@@ -846,9 +839,10 @@ export const visualElement = <Instance, MutableState, Options>({
             isEnabled && restoreTransform(instance, renderState)
         },
 
-        updateLayoutProjection() {
-            isProjecting() && updateLayoutProjection()
```

Comment:
These calls were removed, is that intentional?

Reply:
Yeah this does get called now by the `DepthTree.forEach`